pkg/scheduler/tests/operation_test.go

```diff
@@ -196,7 +196,7 @@ partitions:
 
 	// Check scheduling queue a
 	queueA := part.GetQueue("root.a")
-	assert.Assert(t, 150000000 == queueA.GetMaxResource().Resources[common.Memory])
+	assert.Assert(t, queueA.GetMaxResource().Resources[common.Memory] == 150000000)
```

Comment:
Could you change this to assert.Equal() ?

Reply:
Updated it. I also fix another similar change in recovery_test.go. Thank you.